spacelift/data_stack.go

```diff
@@ -219,10 +234,15 @@ func dataStack() *schema.Resource {
 				Computed:    true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Description: "ID of the GitHub Enterprise application",
+							Computed:    true,
+						},
 						"namespace": {
 							Type:        schema.TypeString,
 							Description: "GitHub Enterprise namespace of the stack's repository",
-							Required:    true,
+							Computed:    true,
```

Comment:
`namespace` shouldn't be required?

Reply:
no, because this is a data source, not a resource.

Reply:
for bitbucket cloud, bitbucket datacenter and azure devops we have required=true as well.

Reply:
And in `data_module.go`, the same.

Reply:
strange, we'll look into it